### PR TITLE
nikolai.berestevich/fix:  bug #21338 text paddings 

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:
-Besquare Title: bottom padding was reduced to 24px 
-What is beSquare ?: text paddings were corrected in  title and subtitles

## Type of change

-   [ x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ x ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/besquare/static/style/_hero.js [hunk 1]
```diff
@@ -34,6 +36,6 @@ export const ImageWrapper = styled.img`
         max-width: 328px;
         margin-right: 15px;
         overflow: unset;
-        top: 40px;
+        top: 15px;
```
Comment (njazuli-deriv) [inline src/pages/besquare/static/style/_hero.js:39]:
```suggestion
        top: 24px;
```

Comment (njazuli-deriv) [reply src/pages/besquare/static/style/_hero.js:39]: Can we change top to 24px as the gap between HeroContainer and ImageWrapper is 24px based on design.

File: src/pages/besquare/static/style/_hero.js [hunk 2]
```diff
@@ -22,6 +21,9 @@ export const HeaderContainer = styled(Container)`
     @media ${device.laptop} {
         padding: 40px 0 0;
     }
+    @media ${device.tablet} {
+        padding: 5px 0 0;
+    }
```
Comment (njazuli-deriv) [inline src/pages/besquare/static/style/_hero.js:26]:
```suggestion
    @media ${device.laptop} {
        padding: 40px 0 0;
    }
```

Comment (njazuli-deriv) [reply src/pages/besquare/static/style/_hero.js:26]: We can removed tablet styles as the margin top for mobile view is also 40px;

**Comment by njazuli-deriv** [comment]
Can we also fixed the padding issue for 'What’s BeSquare?' section as it cause overflow on <330px width @nikolaiberestevich-deriv 
image

File: src/pages/besquare/static/style/_hero.js (hunk 1 above)
Comment (nikolaiberestevich-deriv) [reply src/pages/besquare/static/style/_hero.js]: ok

**Comment by nikolaiberestevich-deriv** [comment]
> Can we also fixed the padding issue for 'What’s BeSquare?' section as it cause overflow on <330px width @nikolaiberestevich-deriv
> image

i did it.

**Comment by nikolaiberestevich-deriv** [comment]
I'll do

**Comment by nikolaiberestevich-deriv** [comment]
i'll do new PR

**Pull request closed without merging**